homeassistant/components/google_cloud/tts.py

```diff
@@ -281,7 +281,7 @@ async def async_get_tts_audio(self, message, language, options=None):
             )
             # pylint: enable=no-member
 
-            with async_timeout.timeout(10, loop=self.hass.loop):
+            with async_timeout.timeout(10):
```

Comment:
Don't you have a warning from async_timeout 4.0, pointing to do a `async with async_timeout...`?

Reply:
Will be fixed in a future turn.  Working on the `loop=` first